lib/utils/KalturaMediaInfo.js

```diff
@@ -0,0 +1,53 @@
+
+const exec = require('child_process').exec;
+const Promise = require('bluebird');
+const fs = require('fs');
+const KalturaMediaInfoResponse = require('./KalturaMediaInfoResponse');
+const COMMAND_LINE = ' -show_streams -show_format -show_programs -v quiet -show_data -print_format json';
+
+/**
+ * class to that pulls mpeg info for every ad file that will be used later
+ */
+class KalturaMediaInfo
+{
+
+    constructor(pathToFFProbe = "ffprobe")

```

Comment:
keep it generic with the ffprobe as default value - call it mediaInfoExecutor or something of that sort